src/locales/ar/translation.json

```diff
@@ -1,5 +1,10 @@
 {
   "title": "[ar] Project Title",
+  "agency": {
+    "longformName": "[ar] NYC Mayor's Office of the Chief Technology Officer",
+    "shortformName": "[ar] NYC CTO"
+  },
+  "nyc": "[ar] NYC",
```

Comment:
A more general property could be used instead of `nyc`, maybe `location`?

Reply:
It's used in the banner for an alt tag, not in the footer. A more general key is a good idea